zerver/lib/import_realm.py

```diff
@@ -1508,6 +1509,15 @@ def do_import_realm(import_dir: Path, subdomain: str, processes: int = 1) -> Rea
     # Realm object is reactivated.
     maybe_enqueue_audit_log_upload(realm)
 
+    # If the export was NOT generated by another zulip server, the
+    # 'zulip_update_announcements_level' is set to None by default.
+    # Set it to the latest level to avoid receiving older update messages.
+    is_realm_imported_from_other_zulip_server = RealmAuditLog.objects.filter(
+        realm=realm, event_type=RealmAuditLog.REALM_EXPORTED, acting_user=None
```

Comment:
I think we should not check `acting_user` here, since exports via the UI can also be imported.